tests/unit/r2i/onnxrt/parameters.cc

```diff
@@ -17,31 +17,6 @@
 #include <CppUTest/CommandLineTestRunner.h>
 #include <CppUTest/TestHarness.h>
 
-namespace mock {
```

Comment:
Why did you remove all these?

Reply:
Solved [here](https://github.com/RidgeRun/r2inference/commit/481b2ed3b0e12d38d57cd863cd25238d368d85f0)